packages/graphql-transformer-core/src/GraphQLTransform.ts

```diff
@@ -331,6 +332,20 @@ export class GraphQLTransform {
     context: TransformerContext,
   ) {
     let index = 0;
+
+    // hashing key directive to get the order in @key transformer
+    let keyDirIndex = 0;
+    let syncKeyMap: Map<string | Int32Array, number> = new Map();
+    if (transformer.name === 'KeyTransformer') {
```

Comment:
I'm not sure we should add the a direct dependency here to KeyTransformer in the the core package. Perhaps we could move the logic back into the key transformer and use the after function to create the final resolver for each model?

Reply:
done